apps/fluent-tester/src/TestComponents/Switch/Switch.tsx

```diff
@@ -18,8 +19,12 @@ const styles = StyleSheet.create({
 });
 
 const StandardUsage: React.FunctionComponent = () => {
+  const memoizedStyles = React.useMemo(
+    () => [Platform.OS === 'android' ? { ...commonTestStyles.androidContainer, height: 180 } : { ...commonTestStyles.settingsPicker }],
```

Comment:
nit: `...commonTestStyles.settingsPicker` no need for the new object or the spread operator here. You can return commonTestStyles.settingsPicker by itself

Reply:
Changed across all the memoized styles.

Reply:
Changes done and approved.